.github/workflows/deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy columbus
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
Current node LTS version 14, may use that one?

Reply:
Gonna change that ! Thank you !